static/source/guides/a_quick_ruby_overview.html.md

```diff
@@ -58,12 +58,12 @@ name += " therox"             # orta therox
 name.start_with? "orta"       # true
 name.end_with? "orta"         # false
 name.gsub "orta", "Orta"      # Orta therox
```

Comment:
That's what I thought but I found that fixing it like this make the reader think harder about what happen.

Thank *you* (and @KrauseFx) for this project !